### PR TITLE
Update CHANGELOG with info about codebeat badge

### PR DESCRIPTION
**Pull request by korzonek**
#### Changes

I've added information about codebeat badge to Misc section of changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Fixes:
 
 Misc:
 - [#1497](https://github.com/rails-api/active_model_serializers/pull/1497) Add JRuby-9000 to appveyor.yml(@corainchicago)
+- [#1551](https://github.com/rails-api/active_model_serializers/pull/1551) Added codebeat badge (@korzonek)
```
Comment (bf4) [inline CHANGELOG.md]: @korzonek would you mind putting the change at the top of the misc list?

Comment (bf4) [inline CHANGELOG.md]: and squashing down to one commit?

**Comment by NullVoxPopuli** [comment]
looks like a rebase needs to happen :-(

**Comment by korzonek** [comment]
@NullVoxPopuli I'm sorry. It should be fine now.

**Comment by NullVoxPopuli** [comment]
woo! now we just need to figure out this travis problem!

thanks for this, and thanks for more code quality tools!

**Comment by NullVoxPopuli** [comment]
I'm gonna merge this, because there is no code change, and master is now passing.

**Pull request merged**